src/Features/Core/Diagnostics/EngineV1/DiagnosticIncrementalAnalyzer.cs

```diff
@@ -24,997 +24,998 @@ namespace Microsoft.CodeAnalysis.Diagnostics.EngineV1
 
     internal partial class DiagnosticIncrementalAnalyzer : BaseDiagnosticIncrementalAnalyzer
     {
-        private static readonly int s_stateTypeCount = Enum.GetNames(typeof(StateType)).Count();
-        private static readonly ImmutableArray<StateType> s_documentScopeStateTypes = ImmutableArray.Create<StateType>(StateType.Syntax, StateType.Document);
+            private static readonly int s_stateTypeCount = Enum.GetNames(typeof(StateType)).Count();
+            private static readonly ImmutableArray<StateType> s_documentScopeStateTypes = ImmutableArray.Create<StateType>(StateType.Syntax, StateType.Document);
```

Comment:
looks like things showed up as many change due to rename/merge conflict.

only change here is this.

return new DiagnosticData(
                    diagnostic.Id,
                    diagnostic.Category,
                    diagnostic.Message,
                    diagnostic.MessageFormat,
                    diagnostic.Severity,
                    diagnostic.DefaultSeverity,
                    diagnostic.IsEnabledByDefault,
                    diagnostic.WarningLevel,
                    diagnostic.CustomTags,
                    diagnostic.Properties, <= added.
                    diagnostic.Workspace,
                    diagnostic.ProjectId,
                    diagnostic.DocumentId,